synapse/lib/view.py

```diff
@@ -421,6 +421,10 @@ async def storm(self, text, opts=None):
         Yields:
             ((str,dict)): Storm messages.
         '''
+        if text is None:
+            mesg = 'storm text cannot be None'
```

Comment:
been slowly eliminating `None` and using `null` to avoid python specific terminology. `Storm query text cannot be null` plz  :+1: